partitioned-table.md

```diff
@@ -753,78 +756,156 @@ In addition, there are limitations on the compatibility of `EXCHANGE PARTITION`
+ PARTITION l2 VALUES IN (2),
+ PARTITION l3 VALUES IN (3),
+ PARTITION l4 VALUES IN (4),
+ PARTITION l5 VALUES IN (5));
+```
 
-PARTITION BY RANGE( YEAR(dob) ) (
-    PARTITION p0 VALUES LESS THAN (1980),
-    PARTITION p1 VALUES LESS THAN (1990),
-    PARTITION p2 VALUES LESS THAN (2000)
-);
+#### Drop partitions
+
+{{< copyable "sql" >}}
+
+```sql
+ALTER TABLE members DROP PARTITION p1990;
+
+ALTER TABLE member_level DROP PARTITION l5;
+```
+
+#### Truncate partitions
+
+{{< copyable "sql" >}}
+
+```sql
+ALTER TABLE members TRUNCATE PARTITION p1980;
+
+ALTER TABLE member_level TRUNCATE PARTITION l4;
+```
+
+#### Add partitions
+
+{{< copyable "sql" >}}
+
+```sql
+ALTER TABLE members ADD PARTITION (PARTITION `p1990to2010` VALUES LESS THAN (2010));
+
+ALTER TABLE member_level ADD PARTITION (PARTITION l5_6 VALUES IN (5,6));
 ```
 
-Drop a partition:
+For Range partitioned table, `ADD PARTITION` can be only appended to the very end of a partition list. If it is appended to an existing Range partition, an error is reported:
 
 {{< copyable "sql" >}}
 
 ```sql
-ALTER TABLE members DROP PARTITION p2;
+ALTER TABLE members ADD PARTITION (PARTITION p1990 VALUES LESS THAN (2000));
 ```
 
 ```
-Query OK, 0 rows affected (0.03 sec)
+ERROR 1493 (HY000): VALUES LESS THAN value must be strictly increasing for each partition
 ```
 
-Empty a partition:
+#### Reorganize partitions
+
+Split a partition:
 
 {{< copyable "sql" >}}
 
 ```sql
-ALTER TABLE members TRUNCATE PARTITION p1;
+ALTER TABLE members REORGANIZE PARTITION `p1990to2010` INTO
+(PARTITION p1990 VALUES LESS THAN (2000),
+ PARTITION p2000 VALUES LESS THAN (2010),
+ PARTITION p2010 VALUES LESS THAN (2020),
+ PARTITION p2020 VALUES LESS THAN (2030),
+ PARTITION pMax VALUES LESS THAN (MAXVALUE));
+
+ALTER TABLE member_level REORGANIZE PARTITION l5_6 INTO
+(PARTITION l5 VALUES IN (5),
+ PARTITION l6 VALUES IN (6));
 ```
 
+Merge partitions:
+
+{{< copyable "sql" >}}
+
+```sql
+ALTER TABLE members REORGANIZE PARTITION pBefore1950,p1950 INTO (PARTITION pBefore1960 VALUES LESS THAN (1960));
+
+ALTER TABLE member_level REORGANIZE PARTITION l1,l2 INTO (PARTITION l1_2 VALUES IN (1,2));
 ```
-Query OK, 0 rows affected (0.03 sec)
+
+Change the partitioning scheme definition:
+
+{{< copyable "sql" >}}
+
+```sql
+ALTER TABLE members REORGANIZE PARTITION pBefore1960,p1960,p1970,p1980,p1990,p2000,p2010,p2020,pMax INTO
+(PARTITION p1800 VALUES LESS THAN (1900),
+ PARTITION p1900 VALUES LESS THAN (2000),
+ PARTITION p2000 VALUES LESS THAN (2100));
+
+ALTER TABLE member_level REORGANIZE PARTITION l1_2,l3,l4,l5,l6 INTO
+(PARTITION lOdd VALUES IN (1,3,5),
+ PARTITION lEven VALUES IN (2,4,6));
 ```
 
-> **Note:**
->
-> `ALTER TABLE ... REORGANIZE PARTITION` is currently unsupported in TiDB.
+Reorganizing partitions (including merging or splitting partitions) changes partitions into a new set of partition definitions. It cannot change the type of partitioning (for example, change the List type to the Range COLUMNS type). 
```

Comment:
According to https://docs.pingcap.com/tidb/stable/partitioned-table#list-columns-partitioning, it seems that we have the List COLUMNS type but do not have the Range COLUMNS type. @mjonss Would you please help check if the "Range COLUMNS type" here is correct? Thanks.